### PR TITLE
Updated advanced_tutorials 100, 200 and 210

### PR DESCRIPTION
**Pull request by hemker**
Updated advanced_tutorials 100, 200 and 210 to match updated basic_tutorials (mainly to compile)

### CODE REVIEW

File: src/documentation/advanced_tutorials/100_deployments.page [hunk 1]
```diff
@@ -5,33 +5,33 @@ sort_info: 100
...
 
 In order to allow a more flexible definition of the execution schema and putting multiple
 tasks into one process, explicit deployments can also be specified. They
-override the default deployment of the task. For further information, also have a look at the
-[documentation page](../orogen/deployment.html).
+override the default deployment of the task. For further information, also have a look on the
+documentation page [here](/documentation/orogen/deployment.html).
 
-Deployments are specified in an orogen project, the same way as tasks are.
+Deployments are specified in an orogen project, the same way than tasks are.
```
Comment (doudou) [inline src/documentation/advanced_tutorials/100_deployments.page]: "same way as" is the correct form http://forum.wordreference.com/showthread.php?t=400069

File: src/documentation/advanced_tutorials/100_deployments.page [hunk 2]
```diff
@@ -5,33 +5,33 @@ sort_info: 100
...
 Even though it is possible to define deployments in the same orogen file as the
-tasks, that is not recommended. Best practice is to create a separate oroGen
+tasks, that is not recommended. Best pratice is to create a separate oroGen
```
Comment (doudou) [inline src/documentation/advanced_tutorials/100_deployments.page]: "practice" not "pratice"

File: src/documentation/advanced_tutorials/100_deployments.page [hunk 3]
```diff
@@ -40,10 +40,10 @@ project (for instance in tutorials/orogen):
 rock-create-orogen messenger_deployments
 ~~~
 
-When asked for dependencies, give the message producer and consumer task projects, e.g.:
+When asked for dependencies give the orogen messages compontent, e.g.:
```
Comment (doudou) [inline src/documentation/advanced_tutorials/100_deployments.page]: "component" not compontent

File: src/documentation/advanced_tutorials/210_data_visualization.page [hunk 4]
```diff
@@ -5,28 +5,28 @@ sort_info: 210
 
 Abstract
 -----------
-In this tutorial, you will learn how to display live and logged data with the help of Vizkit and how to 
+In this tutorial you will learn how to display live and logged data with the help of Vizkit and how to 
 write a custom GUI to display the data of your orocos task.
 
-To understand this tutorial, you must have completed the basic tutorials and the data
+To understand this tutorial you must have competed the basic tutorials and the data
```
Comment (doudou) [inline src/documentation/advanced_tutorials/210_data_visualization.page]: "completed" not "competed"

File: src/documentation/advanced_tutorials/210_data_visualization.page [hunk 5]
```diff
@@ -5,28 +5,28 @@ sort_info: 210
...
 logging tutorial. Basic knowledge about Qt, Qt Designer and qtruby is also
 necessary.
 
-This tutorial directly builds on the previous tutorials. Additionally, you have
+This tutorial directly continues from the last tutorials, additionally, you have
```
Comment (doudou) [inline src/documentation/advanced_tutorials/210_data_visualization.page]: It either "builds on" or "continues" without the 'from'

File: src/documentation/advanced_tutorials/210_data_visualization.page [hunk 6]
```diff
@@ -5,28 +5,28 @@ sort_info: 210
...
 to have the rock packages gui/vizkit and gui/rock_widget_collection installed.
 
-In case you are starting here, the required packages are mentioned on the [introduction page](/documentation/advanced_tutorials/index.html).
+In case you are starting here, the required packages are mentioned on the [introduction page](/documentation/advanced_tutorials/index.html)
 
 Displaying Data with Vizkit
 -----------
-Vizkit is a ruby framework built around qtruby to support the developer with functionality to display his
+Vizkit is a ruby framework build around qtruby to support the developer with functionality to display his
```
Comment (doudou) [inline src/documentation/advanced_tutorials/210_data_visualization.page]: "built" was correct

File: src/documentation/advanced_tutorials/210_data_visualization.page [hunk 7]
```diff
@@ -5,28 +5,28 @@ sort_info: 210
...
 custom data in a convenient way. It is shipped with some Qt Widgets to display some basic types. 
 But its real strength lies in the ability to be customized.
 
-**Displaying Camera Samples** For some base types like the camera frame, Vizkit has a special display widget.
-To create an instance of a widget specialised to display camera frames, you can use the Vizkit ruby 
-method *display* and an Orocos port which is producing camera frames as an argument. In the case below, the camera is 
-substituted by a log file called "bottom_camera.0.log", but you can use the same syntax for Orocos tasks as well.
+**Displaying Camera Samples** For some base types like the camera frame Vizkit has a special display widget.
```
Comment (doudou) [inline src/documentation/advanced_tutorials/210_data_visualization.page]: The commas were definitely correct here.

File: src/documentation/advanced_tutorials/210_data_visualization.page [hunk 8]
```diff
@@ -87,7 +87,7 @@ As you can see, this small script displays the current time and a
 message. Each time the button is pressed, the message is displayed on the command
 line and the time label is updated.
 
-Connecting a custom GUI to an orocos task
+Connecting a custom GUI with an orocos task
```
Comment (doudou) [inline src/documentation/advanced_tutorials/210_data_visualization.page]: People connect with each other. Plugs (or ports) are connected to each other.

File: src/documentation/advanced_tutorials/210_data_visualization.page [hunk 9]
```diff
@@ -146,14 +148,14 @@ We do several things here which require some explanation:
...
  
  * We connect our two labels with the output port of the message producer. 
 
-Connecting a custom GUI to log files
+Connecting a custom GUI with log files
 -----------
-If you want to connect your GUI to a log file, you can use the same method
-as for an orocos task. The following script connects test_gui.ui to a
+If you want to connect your GUI with a log file you can use the same method
+like for an orocos task. The following scripts connects test_gui.ui to a
```
Comment (doudou) [inline src/documentation/advanced_tutorials/210_data_visualization.page]: There's only one script

File: src/documentation/advanced_tutorials/100_deployments.page (hunk 1 above)
Comment (hemker) [inline src/documentation/advanced_tutorials/100_deployments.page]: Hey,

I just altered the pure script parts and don't know why the wrong 
expressions were inserted in the uploaded scripts. Maybe I checked out 
the wrong base/doc branch - I'm sorry for that.

I'm going to fix it at the weekend - my fault.

Greets,
Dennis

Am 14.01.2015 um 19:49 schrieb Sylvain Joyeux:

Comment (doudou) [inline src/documentation/advanced_tutorials/100_deployments.page]: Ah ... OK then ;-)

**Pull request closed without merging**